src/main/java/Duke.java

```diff
@@ -1,10 +1,114 @@
+                    }
+                } catch (NumberFormatException nfe) {
+                    System.out.println(DECO_LINE);
+                    System.out.println("Format is wrong!");
+                    System.out.println(DECO_LINE);
+                }
+                System.out.println(DECO_LINE);
+            } else if (input.split(" ")[0].equals("todo")) {
+                String task = input.split(" ", 2)[1];
+                inputs.add(new Todo(task));
+                System.out.println(DECO_LINE);
+                System.out.println(ADD_TASK);
+                System.out.println("   [T][ ] " + task);
+                System.out.println(" Now you have " + inputs.size() + " tasks in the list.");
+                System.out.println(DECO_LINE);
+            } else if (input.split(" ")[0].equals("deadline")) {
+                try {
+                    String task = input.split(" /by ")[0].split(" ", 2)[1];
+                    String deadline = input.split(" /by ")[1];
+                    inputs.add(new Deadline(task, deadline));
+                    System.out.println(DECO_LINE);
+                    System.out.println(ADD_TASK);
+                    System.out.println("   [D][ ] " + task + " (by: " + deadline + ")");
+                    System.out.println(" Now you have " + inputs.size() + " tasks in the list.");
+                    System.out.println(DECO_LINE);
+                } catch (Exception nfe) {
+                    System.out.println(DECO_LINE);
+                    System.out.println("Format is wrong! You should write \"deadline <event> /by <deadline>\"");
+                    System.out.println(DECO_LINE);
+                }
+            } else if (input.split(" ")[0].equals("event")) {
+                try {
+                    String task = input.split(" /at ")[0].split(" ", 2)[1];
+                    String schedule = input.split(" /at ")[1];
+                    inputs.add(new Event(task, schedule));
+                    System.out.println(DECO_LINE);
+                    System.out.println(ADD_TASK);
+                    System.out.println("   [E][ ] " + task + " (at: " + schedule + ")");
+                    System.out.println(" Now you have " + inputs.size() + " tasks in the list.");
+                    System.out.println(DECO_LINE);
+                } catch (Exception nfe) {
+                    System.out.println(DECO_LINE);
+                    System.out.println("Format is wrong! You should write \"event <event> /at <schedule>\"");
+                    System.out.println(DECO_LINE);
+                }
+            } else {
+                System.out.println(DECO_LINE);
+                System.out.println(ERROR_MESSAGE);
+                System.out.println(DECO_LINE);
+            }
+            input = readinput.nextLine();
+        }
+
+        System.out.println(DECO_LINE);
+        System.out.println(BYE_MESSAGE);
+        System.out.println(DECO_LINE);
```

Comment:
overall, consider applying SLAP principle, it would improve the overall readability of this code.